### DISCUSSION TITLE
Failed to Push some refs to 'https://github.com/username/platform.git'

### DISCUSSION THREAD

**Discussion by ammarumer** [ushahidi/platform#4422] [Outreachy and Google Summer of Code]
**Describe the bug**
In windows 10 when I tried to push a new branch the bug occured.

**Where was the bug observed**
In terminal when tried to push a new branch and commit changes.

**To Reproduce**
1. Go to terminal
2. Create new branch from develop
3. I created update-docs
4. Made changes and committed to branch
5. When tried to push the changes than error occurred and since it was new branch which hasn't been already created it gave the error below.

**Expected behavior**
![image](https://user-images.githubusercontent.com/104008050/164746295-4dcd84e3-6fc1-49f1-9192-983641521c50.png)

**Is there a workaround? What is it.**
Yes I went to the captainhook.json file and then disabled pre-push to false and only then it allowed me to push the branch.
![image](https://user-images.githubusercontent.com/104008050/164746536-358216a9-7a8c-4448-8e80-f28b7ab97f23.png)

![image](https://user-images.githubusercontent.com/104008050/164747615-a6801485-1c20-445e-9937-254bb1ccc359.png)

**Screenshots**

**URL / Environment where this happened**

**If the bug is datasource related (e.g. SMS or Targeted SMS), please specify the configuration details (e.g. Frontline + Clickatell, etc):**

**Desktop Hardware Details (please complete the following information):**
 - OS: Windows 10

**Smartphone Hardware Details (please complete the following information):**

**Additional context**
I've tried using gitbash and all sorts of terminal with latest version but it didn't get fixed that way.

**Comment by tuxpiper** [comment]
@ammarumer we've got restrictions on the repository so only staff can push directly into it. Please make forks for your contributions
